robottelo/datafactory.py

```diff
@@ -179,9 +179,9 @@ def valid_emails_list():
             gen_string('alphanumeric'),
             gen_string('alphanumeric'),
         ),
-        u'"():;"@example.com',
```

Comment:
Why this line was removed?
Also it cause unit-test failing as now this function return less results.